packages/frontend/src/components/projects/sections/contract-entry.tsx

```diff
@@ -166,12 +166,7 @@ function getCalloutProps(
   const isAnyAddressUnverified = contract.addresses.some(
     (c) => c.verificationStatus === 'unverified' && !c.isAdmin,
```

Comment:
but why !c.isAdmin?

Reply:
Removed that flag, replaced with admins array that's simpler and does the same thing